website/content/api-docs/system/config-ui-custom-messages.mdx

```diff
@@ -0,0 +1,242 @@
+---
+layout: api
+page_title: /sys/config/ui/custom-messages - HTTP API
+description: The '/sys/config/ui/custom-messages' endpoint configures the custom messages that the UI displays prior to and immediately following login operations.
+---
+
+# `/sys/config/ui/custom-messages`
+
+The `/sys/config/ui/custom-messages` endpoint is used to list, create, read, update, or delete custom messages.
+
+## List Custom Messages
+
+This endpoint returns a list of all custom messages that exist.
```

Comment:
Should we mention that this endpoint will only return a maximum of 100 messages?

Reply:
Please ignore this comment! I just that was mentioned in the Create section :)